docs/intro-to-coalton.md

```diff
@@ -486,3 +486,9 @@ The following functions all take an optional package parameter
 * `print-value-db` - Print the type of every toplevel value
 * `print-class-db` - Print every class and their methods
 * `print-instance-db` - Print the instances of every class
+
+
+## Coalton quirks
+
```

Comment:
"Lists in Coalton must be homogeneous."

"Coalton uses `fn`, *not* `lambda`, to denote anonymous functions."

"Numerical operators like `+` only take 2 arguments."

"Negation is done with `negate`. The form `(- x)` is a curried function equivalent to `(fn (z) (- x z))`."